docs/updating/migration-to-cdn/testing-suite.md

```diff
@@ -1,5 +1,5 @@
 ---
-menu-title: Testing suite
+menu-title: Vanilla JS testing suite
```

Comment:
That's (not) funny, I already did this change at some point during TP. We are dropping things on the go and don't even realize it.

Reply:
Hmm, this was ok on master, it doesn't show as change rn, as I resolved the conflicts.